incubator/mssql-linux/README.md

```diff
@@ -0,0 +1,126 @@
+
+```console{%raw}
+$ kubectl get all -l app=mssql
+NAME                           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/mymssql-mssql          1         1         1            1           9m
+
+NAME                           DESIRED   CURRENT   READY     AGE
+rs/mymssql-mssql-8688756468   1         1         1         9m
+
+NAME                           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/mymssql-mssql          1         1         1            1           9m
+
+NAME                           DESIRED   CURRENT   READY     AGE
+rs/mymssql-mssql-8688756468   1         1         1         9m
+
+NAME                                 READY     STATUS    RESTARTS   AGE
+po/mymssql-mssql-8688756468-x758g   1/1       Running   0          9m
+
+NAME                 TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+svc/mymssql-mssql   ClusterIP   10.104.152.61   <none>        1433/TCP   9m
+
+```
+
+## Connecting to SQL Server Instance
+1.  Run the following command
+```console
+$ kubectl run dbatoolbox --image=microsoft/mssql-tools -ti --restart=Never --rm=true -- /bin/bash
+$ sqlcmd -S {{ template "mssql.fullname" . }}.{{ .Release.Namespace }} -U sa
+Password: <Enter SA Password>
+```
+
+## Values
+The configuration parameters in this section control the resources requested and utilized by the ZooKeeper ensemble.
+
+| Parameter | Description | Default |
+| --------- | ----------- | ------- |
+| ACCEPT_EULA | EULA that needs to be accepted.  It will need to be changed via commandline or values.yaml. | N |
+| edition | The edition of SQL Server to install | Express |
+| image | |  |
+| repository | The docker hub repo for SQL Server | microsoft/mssql-server-linux |
+| tag | The tag for the image | latest |
+| pullPolicy | The pull policy for the deployment | IfNotPresent |
+| service | | |
+| name | Service Name | mssqlsrvr |
+| type | Service Type | ClusterIP |
+| externalPort | External Port | 1433 |
+| internalPort | Internal Port | 1433 |
+| livenessprobe | | |
+| initialDelaySeconds | Tells the kubelet that it should wait XX second(s) before performing the first probe | 15 |
+| periodSeconds | Field specifies that the kubelet should perform a liveness probe every XX seconds(s) | 20 |
+| readinessprobe | | |
+| initialDelaySeconds | Tells the kubelet that it should wait XX second(s) before performing the first probe | 5 |
+| periodSeconds | Field specifies that the kubelet should perform a liveness probe every XX second(s) | 10 | 
+
+## Resources
+Typically don't recommend specifying default resources and to leave this as a conscious
+choice for the user. This also increases chances charts run on environments with little
+resources, such as Minikube. If you do want to specify resources, uncomment the following
+lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+
+The defaults are set for local development purposes.
+```yaml
```

Comment:
Why not template this also?

Reply:
Can you elaborate on what to put as a template?  It's not clear on the comment.  Thanks.